sim/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name    = "tool-simulation"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
patch versions used, should be changed to major + minor only

Reply:
748d68a6c35bab95af22594b3a08aa94f1788c59